test/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
```

Comment:
Is this where your PigLatin assignment was lost?

Reply:
It's the latest Pig Latin program I did but I am unsure how it ended up on my week1-1 pull.